.github/workflows/build-and-discover.yaml

```diff
@@ -23,9 +23,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
-          pip install chaostoolkit
-          python setup.py develop
+          pip install chaostoolkit .
```

Comment:
Is the full stop supposed to be there?

Reply:
Yep! 

`pip install .` says 'install the package that is here', as `chaostoolkit-aws` is an actual package in itself, it'l install itself into the `venv`

Reply:
`python setup.py develop` does essentially the same thing 😄 It's actually not as useful as we don't need to install the package for development, we're just installing it to use it